readme.md

```diff
@@ -4,7 +4,7 @@
 
 ### early alpha - subject to breakage, changes, argument, refactors
 
-_Try it in your browser [right now](https://iodide-project.github.io/master/)!_
+_Try it in your browser [right now](https://iodide-project.github.io/master/?iodide-example=what-a-web-notebook-looks-like.html)!_
```

Comment:
hmm, this is a bit tricky. I completely agree that it'd be much preferable to serve a url directly, but i'm not sure how to do it at the moment-- for example, the notebook i want to share is the one located at:
https://github.com/iodide-project/iodide-examples/blob/master/what-a-web-notebook-looks-like.html
but to make committing it expedient, i just committed the local version i'd saved in my `dev/` folder with relative links to dev resources, e.g. `<script src='iodide.dev.js'></script>`. So serving that file directly wouldn't work because the resource paths+names are wrong for web deployment, and in any case i really want to load the jsmd in that notebook using Iodide master.

We could do this much better by having some kind of better build process for notebooks in iodide-examples (something that always points at the latest version, or maybe for now at master?). That's for sure where we should go in the long run. So i just consider this an ugly hacky shortcut that I 100% agree we want to replace as soon as we can come up with a nicer workflow.

Nonetheless, i think it's the shortest path to dropping users in an informative notebook rather than a blank notebook, so i'm willing to live with it for now, and we can file an issue around improving the workflow for iodide-examples and serving notebooks as html.

what do you think @wlach, does that sound ok for now?

Reply:
Hmm, I don't quite understand why `?url=https://raw.githubusercontent.com/iodide-project/iodide-examples/master/what-a-web-notebook-looks-like.html` won't work? It seems to load fine for me locally. What do you mean the resource paths and names are wrong?

Reply:
Oh i see what you mean, sorry! Yes we can use that if you think it's better-- I'll make the change.

Reply:
you're right, i'll remove the `iodide-examples` option